slurm-mlr/_drake.R

```diff
@@ -0,0 +1,21 @@
+# Load packages and function -----------------------------------------------------------
+source("packages.R")
+
+sourceDirectory("R")
+
+# Set Slurm options for workers -------------------------------------------
+
+options(clustermq.scheduler = "slurm",
+        clustermq.template = "slurm_clustermq.tmpl")
+
+# Create plans ------------------------------------------------------------
+
+benchmark_plan = code_to_plan("benchmark.R")
```

Comment:
Cool application of `code_to_plan()`!

Reply:
Usually I have it even more fine-grained to align plans with "stages" specified in `_drake.R` which are then also used for visualizing the graph.